lib/db/files_db.dart

```diff
@@ -7,14 +7,15 @@ import 'package:photos/models/file.dart';
 import 'package:path/path.dart';
 import 'package:sqflite/sqflite.dart';
 import 'package:path_provider/path_provider.dart';
+import 'package:sqflite_migration/sqflite_migration.dart';
 
 class FilesDB {
   static final _databaseName = "ente.files.db";
-  static final _databaseVersion = 1;
```

Comment:
How is versioning maintained now?

Reply:
It is done by the sqflite_migration library based on the number of migration scripts present.